src/components/Profile/Profile.js

```diff
@@ -3,10 +3,15 @@ import React from 'react';
 import MyPosts from './MyPosts/MyPosts';
 import ProfileInfo from './ProfileInfo/ProfileInfo';
 
-const Profile = () => (
+const Profile = (props) => (
   <div>
     <ProfileInfo />
-    <MyPosts />
+    <MyPosts
+      posts={props.profilePage.posts}
```

Comment:
It's too much of repetition of "props".
Consider adding destructuring of props.

Reply:
Soon it will all be replaced